app/jobs/cache/feed_job.rb

```diff
@@ -41,7 +42,7 @@ def after_perform
     if @updated
```

Comment:
1. Could we change this to `return unless @updated`?
2. I reckon we should also only enqueue the `DownloadRemoteMediaObjectJob` jobs if the feed was updated, so the code to handle that from the end of `#perform` could be moved into the `after_perform` callback.